articles/azure-monitor/app/correlation.md

```diff
@@ -73,6 +73,10 @@ We're transitioning to [W3C distributed tracing format](https://w3c.github.io/tr
 - `tracestate`: Carries tracing system-specific context.
 
 #### Enable W3C distributed tracing support for classic ASP.NET apps
```

Comment:
lines 59 and 66 has outdates links to Request-Id thing. its good to update those old links.

Reply:
they seem to be the latest available links - do you know about better ones?

Reply:
i meant this one actually: https://github.com/lmolkova/correlation/blob/master/http_protocol_proposal_v2.md

Reply:
also "We're working on an RFC proposal for the [correlation HTTP protocol]" - change this to say something like this is already existing in current sdks before w3c.

Reply:
> https://github.com/lmolkova/correlation/blob/master/http_protocol_proposal_v2.md

even though it does not look legit, there is no better spec

Reply:
okay. got it.